js/coinbase.js

```diff
@@ -572,6 +572,9 @@ module.exports = class coinbase extends Exchange {
                                 'min': this.safeNumber (quoteCurrency, 'min_size'),
                                 'max': undefined,
                             },
+                            'leverage': {
+                                'max': 1,
```

Comment:
i don't think we should define this value here...

Reply:
why